checkov/terraform/checks/resource/kubernetes/DropCapabilities.py

```diff
@@ -0,0 +1,54 @@
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class DropCapabilities(BaseResourceCheck):
+
+    def __init__(self):
+        # CIS-1.3 1.7.7
+        # CIS-1.5 5.2.7
+        # NET_RAW allows a process to spy on packets on its network
+        name = "Minimize the admission of containers with the NET_RAW capability"
+        id = "CKV_K8S_28"
+
+        supported_resources = ['kubernetes_pod']
+        categories = [CheckCategories.GENERAL_SECURITY]
```

Comment:
why not `CheckCategories.KUBERNETES`?

Reply:
as every check for the provider would be kubernetes?

Reply:
I don't know actually, do we have some guidelines for it?

Reply:
if we have I dont know about it.